lms/djangoapps/verify_student/views.py

```diff
@@ -1154,16 +1154,10 @@ def results_callback(request):
     except SoftwareSecurePhotoVerification.DoesNotExist:
         log.error("Software Secure posted back for receipt_id %s, but not found", receipt_id)
         return HttpResponseBadRequest("edX ID {} not found".format(receipt_id))
-    context = {'user': request.user}
     if result == "PASS":
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
-        context['subject'] = _("Your {platform_name} ID Verification Approved").format(
-            platform_name=settings.PLATFORM_NAME
-        )
-        context['message'] = 'emails/successfull_verification_email.txt'
-        send_verification_status_email(context)
```

Comment:
We need to remove the import for this method from this file for failing tests.

Reply:
Done